PWGJE/EMCALJetTasks/UserTasks/AliAnalysisTaskJetPP.cxx

```diff
@@ -528,7 +532,10 @@ void AliAnalysisTaskJetPP::UserCreateOutputObjects(){
    fOutput->Add((TH1D*)fhJetPt);
    fOutput->Add((TH1D*)fhJetPtRho);
    fOutput->Add((TH1D*)fhJetPtConeRho);
-   fOutput->Add((TH1D*)fhAtimesRho);
+   fOutput->Add((TH1D*)fhJetPtCMSRho);
```

Comment:
These variables are already of the THXD*, so the cast (TH1D*) and (TH2D*) is redundant and can be removed.